doc/host_config_schema/cchost_config.json

```diff
@@ -467,6 +484,65 @@
       },
       "description": "This section includes configuration for the host-enclave ring-buffer memory (modify with care!)",
       "additionalProperties": false
+    },
+    "acme": {
+      "type": "object",
+      "properties": {
+        "configurations": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "object",
+            "properties": {
+              "ca_certs": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "default": [],
+                "description": "Root certificate(s) of the CA to connect to in PEM format (for TLS connections to the CA, e.g. Let's Encrypt's ISRG Root X1)"
+              },
+              "directory_url": {
+                "type": "string",
+                "default": "",
+                "description": "URL of the ACME server's directory"
+              },
+              "service_dns_name": {
+                "type": "string",
+                "default": "",
+                "description": "DNS name of the service we represent"
+              },
+              "contact": {
+                "type": "array",
+                "items": {
+                  "type": "string"
+                },
+                "default": [],
+                "description": "Contact addresses (see RFC8555 7.3, e.g. mailto:john@example.com)"
+              },
+              "terms_of_service_agreed": {
+                "type": "boolean",
+                "default": false,
+                "description": "Indication that the user/operator is aware of the latest terms and conditions for the CA"
+              },
+              "challenge_type": {
+                "type": "string",
+                "default": "http-01",
+                "description": "Type of the ACME challenge (currently only http-01 supported)"
+              }
+            },
+            "description": "ACME Configuration",
+            "additionalProperties": false
+          },
+          "description": "ACME Configurations"
+        },
+        "challenge_server_interface": {
+          "type": "string",
+          "default": "0.0.0.0:80",
```

Comment:
I don't think we should impose a default here. However, we should mention in the description that this interface should be accessible publicly via port 80.

Reply:
This is just a default value, which will work for most systems I think. Note that the port 80 restriction only applies to Let's Encrypt's services, but not in general. E.g. the pebble CA for testing allows us to use other ports so that we don't need system permissions for port 80. So, we can give a default, but we can't throw a hard error if someone choses a different port.